.circleci/config.yml

```
@@ -533,8 +533,17 @@ jobs:
       - run:
           name: "Run Tests: JavaScript Tests"
           command: node ./scripts/run-ci-javascript-tests.js --maxWorkers 2
-      - run_e2e:
-          platform: js
+
+      # When not in 0.XX-stable branch, run e2e tests
+      - when:
+          condition:
+            not:
+              matches:
+                pattern: "0.[0-9]{2}-stable"
+                value: << pipeline.git.branch >>
+          steps:
+            - run_e2e:
+                platform: js
```

```suggestion

      # When not in 0.XX-stable branch, run e2e tests
      - when:
          condition:
            not:
              matches:
                pattern: "0.[0-9]{2}-stable"
                value: << pipeline.git.branch >>
          steps:
            - run_e2e:
                platform: js
```

From @ryancat's comments it looks like the test we were wanting to disable might still be running. This change originally came from the main branch to conditionally run on stable branches.

I wonder if `<< pipeline.git.branch >>` might evaluate to the branch a PR comes from or some jazz like that. But if this is only to the single branch we could also just remove everything.

FYI @ryancat, since this change targets a "stable branch" instead of "main", we do not sync it to Phabricator, and this change can be committed directly in GitHub.

---

@NickGerleman the test is still running on this PR because the branch is `kelset/filter-e2e-on-071` ;)